### PR TITLE
Add physics bridge

### PR DESCRIPTION
**Pull request by nik0la31**

### CODE REVIEW

File: MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs [hunk 1]
```diff
@@ -155,6 +157,7 @@ internal MixedRealityExtensionApp(string globalAppId, MonoBehaviour ownerScript,
 			_assetLoader = new AssetLoader(ownerScript, this);
 			_userManager = new UserManager(this);
 			_actorManager = new ActorManager(this);
+			_physicsBridge = new PhysicsBridge();
```
Comment (jabenk) [inline MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs:160]: Might we need here "this" for the physics bridge ?

File: MREUnityRuntime/MREUnityRuntimeLib/Core/PhysicsBridge.cs [hunk 2]
```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MixedRealityExtension.Core
+{
+	public class PhysicsBridge
```
Comment (jabenk) [inline MREUnityRuntime/MREUnityRuntimeLib/Core/PhysicsBridge.cs:9]: I think some code comments would be usefull

Comment (jabenk) [reply MREUnityRuntime/MREUnityRuntimeLib/Core/PhysicsBridge.cs:9]: on the functions level and maybe not for all functions

File: MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs (hunk 1 above)
Comment (nik0la31) [reply MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs:160]: There was not need for it yet. We may add it later.

File: MREUnityRuntime/MREUnityRuntimeLib/Core/PhysicsBridge.cs (hunk 2 above)
Comment (nik0la31) [reply MREUnityRuntime/MREUnityRuntimeLib/Core/PhysicsBridge.cs:9]: Agree, that's a big todo.

**Comment by nik0la31** [comment]
Added initial physics bridge classes.

**Pull request merged**